arci-speak-audio/Cargo.toml

```diff
@@ -16,6 +16,6 @@ tracing = { version = "0.1", features = ["log"] }
 tokio = { version = "1", features = ["sync", "parking_lot"] }
 
 [dev-dependencies]
-clap = { version = "3.1", features = ["derive"] }
+clap = { version = "~3.1", features = ["derive"] }
```

Comment:
This can now be reverted as deprecations are now off by default, see clap-rs/clap#3830 for details.  When you do start working through the deprecations as part of upgrading to 4.0, we've also improved the warnings for derive users.  See clap-rs/clap#3832 for examples.